sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/ShowTablesSuite.scala

```diff
@@ -34,7 +35,7 @@ trait ShowTablesSuiteBase extends command.ShowTablesSuiteBase {
   override def defaultNamespace: Seq[String] = Seq("default")
   override def showSchema: StructType = {
```

Comment:
seems like we don't need this API anymore, as the schema is unified. Can you clean up this test suite (and parent suites)?

Reply:
OK